gnat2goto/driver/goto_utils.adb

```diff
@@ -18,6 +18,93 @@ package body GOTO_Utils is
       return Size_T;
    end CProver_Size_T;
 
+   Float_T : Irep := Ireps.Empty;
```

Comment:
These are all a bit misleading as the C standard doesn't really guarantee any of these sizes and it's not like we're getting them from an actual C compiler

Reply:
C standard doesn't but IEEE-754 does and there is a link between the Ada standard and it.

Reply:
Ah, scratch that.  Yes, it's less clear what is going on and why this is needed.  ISO 9899 gives minimum ranges for each type but not their actual width.  I would be more comfortable, if you need C compatible things, with using `int64_t` et al.

Reply:
renamed.

Reply:
Thank you.